src/main/resources/db/migration/V2024_06_21_1155__CIV-14169-update-task-list-final-orders-issued.sql

```diff
@@ -0,0 +1,17 @@
+/**
+ * Add scenario
+ */
+INSERT INTO dbs.scenario (name, notifications_to_delete, notifications_to_create)
+VALUES ('Scenario.AAA6.Update.TaskList.UploadDocuments.FinalOrders', '{}', '{}');
```

Comment:
You'll also need to trigger the notifications that are triggered as part of  SCENARIO_AAA6_CP_ORDER_MADE_DEFENDANT

Reply:
I have choose different approach at the end, please have a look.